src/api/objects/ObjectAPI.js

```diff
@@ -100,7 +100,8 @@ export default class ObjectAPI {
       'restricted-notebook',
       'plan',
       'annotation',
-      'activity-states'
+      'activity-states',
+      'plan-execution-monitoring'
```

Comment:
Do we want to use PLAN_EXECUTION_MONITORING_KEY here?

Reply:
LOL! Yes, we do!